ptsprojects/zephyr/l2cap.py

```diff
@@ -294,14 +294,25 @@ def test_cases(pts):
                   pre_conditions_eatt,
                   generic_wid_hdl=l2cap_wid_hdl),
         ZTestCase("L2CAP", "L2CAP/ECFC/BV-11-C",
-                  # this test requires EATT enabled, don't create L2CAP server
-                  common,
+                  common +
+                  [TestFunc(lambda: pts.update_pixit_param("L2CAP", "TSPX_iut_supported_max_channels", "2")),
```

Comment:
I would extract common code and create a new list similar to `pre_conditions_eatt`.

Reply:
Done :)